### DISCUSSION TITLE
Generic OAuth2 & high traffic

### DISCUSSION THREAD

**Discussion by braco** [outline/outline#4827] [General] [answered]
Hi, two questions. I run a fairly high traffic subreddit, I'm looking to bring that audience to a separate domain.

1. Could this wiki support Reddit's OAuth2? That kind of feature wasn't clear from the docs. Wiki.js seems to support this but I prefer your stack.

2. Outline seems oriented toward corporate teams based on the messaging on your site. Does this wiki make sense for a public-facing, potentially high traffic wiki? The edits would be infrequent but the traffic might be heavy.

Thanks :)

**Comment by tommoor** [comment, marked as answer]
> Could this wiki support Reddit's OAuth2? That kind of feature wasn't clear from the docs. Wiki.js seems to support this but I prefer your stack.

If it's OIDC compatible then yes, whether this is the case I don't know.

> Outline seems oriented toward corporate teams based on the messaging on your site. Does this wiki make sense for a public-facing, potentially high traffic wiki? 

It's optimized for searchable dynamic content in an authenticated environment, the pages are not statically generated or cached like you might expect on a public-facing wiki. That said, it can scale horizontally to handle high traffic without issue.
